### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by bizip**
- Add RSpec as a gem to your project.

- Create a Request spec file for all your controllers.

- Check if the response status was correct.

-  Check if a correct template was rendered.

- Check if the response body includes the correct placeholder text.

### CODE REVIEW

File: spec/requests/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
+  before(:each) do
+    @user = User.create(
+      name: 'Pascal',
+      photo: 'https://images.unsplash.com/photo-1660911295696-9810c266d9f2?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG
```
Comment (mikethreels) [inline spec/requests/post_spec.rb]: - [x] according to the requirements you should use type: request

File: spec/requests/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```
Comment (mikethreels) [inline spec/requests/user_spec.rb]: - [ ] according to the requirements you should use type: request

File: spec/requests/post_spec.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    end
+  end
+
+  describe 'GET show Pasges' do
+    it 'It should renders show page' do
+      get :show, params: { id: @post.id, user_id: @user.id }
+      expect(response.status).to eq(200)
+      expect(response).to render_template('show')
+    end
+  end
```
Comment (mikethreels) [inline spec/requests/post_spec.rb]: - [ ] The Spec file should include tests that ensure the response body includes the correct placeholder text

File: spec/requests/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    end
+  end
+
+  describe 'GET tthe show show page' do
+    it 'renders the show template' do
+      get :show, params: { id: @user.id }
+      expect(response.status).to eq(200)
+      expect(response).to render_template('show')
+    end
+  end
```
Comment (mikethreels) [inline spec/requests/user_spec.rb]: - [ ] The Spec file should include tests that ensures the response body includes the correct placeholder text

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Pascal, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by descholar-ceo — APPROVED** [review]
Hi @bizip 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Implemented the requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**